kuma/static/js/components/local-anchor.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Find all level two and three headings, and append a link
+ * element that links directly to the heading
+ */
+function appendLocalAnchors() {
```

Comment:
I have no idea why this reads `unction` :) My local file definitely does not.

Reply:
Oh, and if I click on the view button, the resulting view does show `function`. Guess it's a Github issue.

Reply:
It looks like ``function`` to me. Update or uninstall those plugins!

Does ``appendLocalAnchors`` mean HTML element order? The link appears to prepend the headings for me.

Reply:
Good call, `prependLocalAnchors` makes more sense. I was initially appending, and did not think to change the name when I switched to prepending ;)